content/chainguard/chainguard-images/how-to-use/use-chainguard-helm-charts.md

```diff
@@ -0,0 +1,233 @@
+
+    Configuring identity "45a0c61ea6fd977f050c5fb9ac06a69eed764595/095b0c7ea9d68679" for pulls from cgr.dev (expires 2025-06-12T09:27:45-05:00).
+    Overwriting existing credentials.
+    ```
+
+    Save the credentials as variables, like this.
+
+    ```sh
+    HELMUSER=45a0c61ea6fd977f050c5fb9ac06a69eed764595/095b0c7ea9d68679
+
+    HELMPASS=eyJhbGciOiJSUzI1NiJ9.eyJhdWQiOiJodHRwczovL2lzc3Vlci5lbmZvcmNlLmRldiIsImV4cCI6MTc0OTczODQ2NSwiaWF0IjoxNzQ5NjUyMDY2LCJpc3MiOiJodHRwczovL3B1bGx0b2tlbi5pc3N1ZXIuY2hhaW5ndWFyZC5kZXYiLCJzdWIiOiJwdWxsLXRva2VuLTAxY2MwODkwYzA5N2ZmMzk1MDUyMWY4NWFmYmEyZDUwMGM0ODQxOWEifQ.ET7ywPUkMk5wN6p0INqhNtdnOVELySqdjp-qWedVmJkLrWlZhdFodU43P4uuR-LJ3Z9mVmd9fjDWpBtZnsCFHbczkENPzOiAFP9fsJhO_2dXT3rXCPK84ddJgRLe6oDlMA3VSa0XEclfTyBcaG4RlrgkVaGhtS7gone4Egff7bKX5Y6-TUxxLiVvCA_l_YmOixUss_Mj1Qxxb81sCeh7x4FSpOGWtmU2Z7Hy6B_rGk17zXMO_GYcuyzAMxfFdQl1Ov18t7KxymQwIoS7UF1fx_5ECR8fgArLM8NikGOjzkiQZuSzeI_hl_GnUFdPTAAhmjpJEWO0isiSPWgpkUPx5scoSUm6jzfduvRgGcmjRxT_pq6MWzFJNw9gv9gVehJuW5lKzNIgMTfJXO5Roba8WCwwxiUknhZXP8DeD_kdAN2-JbkfOYg3aPVU5jFTtA6TJKlh0uQA5OGN5hG_PnyzIr0vu4VVninJTWm66RppdlffhG-1xY9lpXgD2k2TIhygFL8iEBNszq0siLVA3uTH6NZY8iGRFqziUAGnyD80aHn52tIeCBBAOyS6qfcRLzqO6dQX95uscdCOuy-5rxU9n4208m5duLXdZtVWa9gp2vg-OmxnCPVdXmPCTA6RF43gDVkxKGMfvkUkTW1nKNvIUx_ikC9tLHDuZdi8FKLeYEg
+    ```
+
+3. Create the Kubernetes secret using the variables you just created.	
+
+    ```sh
+    kubectl create secret docker-registry chainguard-pull-secret \
+      --docker-server=cgr.dev \
+      --docker-username=$(echo $HELMUSER) \
+      --docker-password=$(echo $HELMPASS) \
+      -n <your-namespace>
+    ```
+
+4. Log in to the `cgr.dev` Helm registry
+
+    ```sh
+    helm registry login cgr.dev \
+      --username=$HELMUSER \
+      --password=$HELMPASS
+    ```
+
+5. Reference the secret in your Helm installation
+
+    ```sh
+    helm install rabbitmq oci://cgr.dev/$ORGANIZATION/iamguarded-charts/rabbitmq \
+      --set "global.org=$ORGANIZATION" \
+      --set "global.imagePullSecrets[0].name=chainguard-pull-secret"
+    ```
+
+    When successful, you will see something like this:
```

Comment:
Should this be intended under point 5? Is the "successful" referencing only step 5 (because that's what I'd assume readign this)?